tests/console/validate_modify_existing_partition.pm

```diff
@@ -0,0 +1,39 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Verify the partition modified in modify_existing_partition.
+# Maintainer: Jonathan Rivrain <jrivrain@suse.com>
+
+use strict;
+use warnings;
+use base "opensusebasetest";
+use testapi;
+use scheduler 'get_test_data';
+use Test::Assert ':all';
+
+sub run {
+    my $test_data  = get_test_data();
+    my $fs_type    = $test_data->{fs_type};
+    my $partition  = $test_data->{existing_partition};
+    my $mountpoint = $test_data->{mount_point};
+    my $part_size  = $test_data->{part_size};
+    $part_size =~ s/[A-Za-z]//g;
+
+    select_console "root-console";
+    record_info("Check $fs_type", "Verify that the partition filesystem is $fs_type");
+    my $fstype = script_output("df -PT $mountpoint | grep -v \"Filesystem\" | awk '{print \$2}'");
```

Comment:
```suggestion
    my $fstype = script_output("blkid /dev/sda2 | awk -F'[ ,=]' '{print $7}'');
```

Reply:
That would be shorter, but you would need to add full path to partition in `test_data` (feel free to leave it as it).

Reply:
Does not work, this does blkid /dev/sda2 | awk -F'[ ,=]' '{print $5}' |tr -d \". Probably some better awk regex would do, but I'm lazy to find it out, I'll leave it as it is now.

Reply:
it should work, I meant `/dev/vda2`. I wrote by mistake `/dev/sda2` because I was trying in my laptop (but it is not important)